crates/symbolicator-proguard/src/metrics.rs

```diff
@@ -17,7 +17,7 @@ pub fn record_symbolication_metrics(
         .map(|p| p.as_ref())
         .unwrap_or("none");
 
-    metric!(time_raw("symbolication.num_exceptions") = exceptions.len() as u64);
+    metric!(time_raw("symbolication.num_exceptions") = exceptions.len() as u64, "event_platform" => event_platform);
```

Comment:
Should we map `Platform::Other(_)` to "other" here? Or are we confident that sentry will not generate more than a handful of platform names?

Reply:
That appears to be the case, there are only two or three that I've seen other than the hard-coded ones.